### ISSUE TITLE
Widget in 3D space

### ISSUE THREAD

**Issue by devildevilson** [Immediate-Mode-UI/Nuklear#657]
Nuklear is wonderful gui. What can makes it more wonderful? The possibility to use it as info billboard on top of a unit. Is there a way? For example 
![6Ak1A](https://github.com/Immediate-Mode-UI/Nuklear/assets/50243012/e226f1fc-4343-461e-903a-ac48dcc4be6c)
(not my screenshot, found in the internet)
Is there a way to render text on top of theese guys using Nuklear? I beleve it is needed to pass some 3d matrix to every nuklear window and catch it in nk_convert... Something like that. How can I deal with it?
PS: if its already answered, Im sorry - google gives nothing

**Comment by RobLoach** [comment]
Yes, you would likely apply a transform before rendering the contexts. Depends on which renderer you're using though.

**Comment by devildevilson** [comment]
Yep, Ill write my own renderer like no problem. I need to pass some additional info into the nuklear window. How to do it? I dont see how to pass userdata around window in nuklear

**Comment by RobLoach** [comment]
`nk_set_user_data()`? Make sure to define NK_INCLUDE_COMMAND_USERDATA.
